buildbot/buildbot/state.py

```diff
@@ -1,4 +1,26 @@
-# These are the states that jobs can be in.
+# These are the states that jobs can be in. The transitions are:
+#
+#
+#       UPLOAD
+#         |
+#       UNPACK---------------+
+#         |                  |
+#    UNPACK_FINISH           |
+#         |                MAKE
+#       COMPILE              |
+#         |                  |
+#    COMPILE_FINISH          |
+#         |                MAKE_PROGRESS
+#        HLS                 |
+#         |                  |
+#      HLS_FINISH            |
+#         |                  |
+#        RUN-----------------+
```

Comment:
Seems like this "edge" should connect to `HLS_FINISH`.

Reply:
Fixed